README.md

```diff
@@ -1,43 +1,419 @@
+The latest available version is: 153.0.0
+
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│                                                  Components                                                 │
+├───────────────┬──────────────────────────────────────────────────────┬──────────────────────────┬───────────┤
+│     Status    │                         Name                         │            ID            │    Size   │
+├───────────────┼──────────────────────────────────────────────────────┼──────────────────────────┼───────────┤
+│ Not Installed │ App Engine Go Extensions                             │ app-engine-go            │  47.9 MiB │
+│ Not Installed │ Bigtable Command Line Tool                           │ cbt                      │   3.9 MiB │
+│ Not Installed │ Cloud Datalab Command Line Tool                      │ datalab                  │   < 1 MiB │
+│ Not Installed │ Cloud Datastore Emulator                             │ cloud-datastore-emulator │  15.4 MiB │
+│ Not Installed │ Cloud Datastore Emulator (Legacy)                    │ gcd-emulator             │  38.1 MiB │
+│ Not Installed │ Cloud Pub/Sub Emulator                               │ pubsub-emulator          │  21.0 MiB │
+│ Not Installed │ Emulator Reverse Proxy                               │ emulator-reverse-proxy   │  14.5 MiB │
+│ Not Installed │ Google Container Registry's Docker credential helper │ docker-credential-gcr    │   3.4 MiB │
+│ Not Installed │ gcloud Alpha Commands                                │ alpha                    │   < 1 MiB │
+│ Not Installed │ gcloud Beta Commands                                 │ beta                     │   < 1 MiB │
+│ Not Installed │ gcloud app Java Extensions                           │ app-engine-java          │ 128.6 MiB │
+│ Not Installed │ gcloud app Python Extensions                         │ app-engine-python        │   6.1 MiB │
+│ Not Installed │ kubectl                                              │ kubectl                  │  14.9 MiB │
+│ Installed     │ BigQuery Command Line Tool                           │ bq                       │   < 1 MiB │
+│ Installed     │ Cloud SDK Core Libraries                             │ core                     │   5.9 MiB │
+│ Installed     │ Cloud Storage Command Line Tool                      │ gsutil                   │   2.9 MiB │
+│ Installed     │ Default set of gcloud commands                       │ gcloud                   │           │
+└───────────────┴──────────────────────────────────────────────────────┴──────────────────────────┴───────────┘
+```
+
+For example, for python GAE support, you would need to extend the base image:
+
+```dockerfile
+FROM google/cloud-sdk
+RUN gcloud components install app-engine-python
+```
+
+Then build and run that:
+```
+docker build -t cloud-sdk-docker-gae .
+docker run -t cloud-sdk-docker-gae
+```
+
+
+### Upgrading SDK versions
```

Comment:
incase somene would fork and want to manage cloudsdk image for their internal company repo.  its instructions on build.  could move this to another readme since the root one goes to dockerhub